infrastructure/local/peer-docker-compose.yml

```diff
@@ -69,3 +70,6 @@ services:
       - ./seed.peer.yml:/workspace/seed.peer.yml
     depends_on:
       - peer-backend
+networks:
+  local_rafiki:
+    external: true
```

Comment:
I added this after noticing :point_down: while testing:
```
$ docker compose -f infrastructure/local/peer-docker-compose.yml ps
service "peer-auth" refers to undefined network rafiki: invalid compose project
```
I am now sometimes getting the following when starting the localenv:
```
network local_rafiki declared as external, but could not be found
```
I wonder if that can be avoided by splitting these into separate/subsequent `docker compose` commands:
https://github.com/interledger/rafiki/blob/632bdfb15a920062637dc8a273034e58b382205b/package.json#L20-L21